src/Conversion/InterventionImageFileConverter.php

```diff
@@ -90,21 +93,26 @@ private function validateOptions(array $options): array
     private function supportedByIntervention(string $format, InterventionBackend $backend): bool
     {
         $driver = $backend->getImageManager()->config['driver'] ?? null;
-        // If the driver is somehow not GD or Imagick, we have no way to know what it might support
-        if ($driver !== 'gd' && $driver !== 'imagick') {
-            return false;
-        }
 
         // Return early for empty values - we obviously can't support that
         if ($format === '') {
             return false;
         }
 
+        $format = strtolower($format);
+
+        // If the driver is somehow not GD or Imagick, we have no way to know what it might support
+        if ($driver !== 'gd' && $driver !== 'imagick') {
+            $supported = false;
+            $this->extend('updateSupportedByIntervention', $format, $supported, $driver);
```

Comment:
```suggestion
            $this->extend('updateSupportedByIntervention', $supported, $format, $driver);
```
Change around the params to make it a little more intentful what this is for

We're updated $supported so I think it needs to be the first param as in convention

I think $format is probably next important because it most cases there's probably only one third party library, so won't need to if/else it, so it's less important that $format.

Reply:
The order of params for extension hooks isn't mentioned in [our coding conventions](https://docs.silverstripe.org/en/5/contributing/coding_conventions/php_coding_conventions/#extension-hooks)... maybe it should be?
In this case I don't have any problem changing this order so I'll do that.

Reply:
Done